index.d.ts

```diff
@@ -1,13 +1,14 @@
 import { Context } from "aws-lambda";
-import { FastifyInstance, FastifyReply } from "fastify";
+import { FastifyInstance } from "fastify";
+import {Response as LightMyRequestResponse} from "light-my-request";
 
```

Comment:
Fastify exports LightMyRequestResponse already

```suggestion
import { FastifyInstance, LightMyRequestResponse } from "fastify";
```